yaml-tests/src/test/resources/aggregate-index-tests.yamsql

```diff
@@ -162,9 +162,6 @@ test_block:
     -
        - query: select max(col2) from T1 use index (mv8);
        - maxRows: 1
-       # Cannot use FORCE_CONTINUATIONS with older versions due to: https://github.com/FoundationDB/fdb-record-layer/issues/3096
-       # (Extra values being produced after exhausting source of an aggregate cursor)
-       # Can remove once we do not care about backwards compatibility before 4.1.9.0
```

Comment:
I think we either want to retain this comment or remove the conditional behavior. Otherwise, it'll be unclear to us why we have the version check when we look at this.

At present, it actually looks like we're no longer testing against pre-4.1.9.0 versions in our mixed mode testing. So we could probably just remove this conditional behavior. In fact, it looks like this is actually the same query as the one before, and we could probably clean this up by: (1) removing this test case entirely and (2) removing the `supported_version` directive from the previous query